.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Please include a summary of the change and which issue is fixed or which feature is added.
+
+- [ ] Issue 1 fixed
+- [ ] Issue 2 fixed
+- [ ] Feature 1 added
+- [ ] Feature 2 added
+
+Fixes # (issue)
+
+# How to test this?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce if there is no integration test added with this PR. Please also list any relevant details for your test configuration
+
+"""
```

Comment:
Formatting here may be off. How about using: ```bash  ....

Reply:
Good catch! I got confused by all the different ways to add code snippets in other platforms. Thank you!